target/linux/ramips/dts/VOCORE2.dts

```diff
@@ -1,45 +1,18 @@
 /dts-v1/;
 
-#include "mt7628an.dtsi"
-
-#include <dt-bindings/gpio/gpio.h>
-#include <dt-bindings/input/input.h>
+#include "VOCORE2.dtsi"
 
 / {
-	compatible = "vocore,vocore2", "mediatek,mt7628an-soc";
```

Comment:
please keep the compatible string here. I know it is inherit from the dts, but I feel we should have it anyway.